src/main/java/io/openapitools/api/capabilities/Element.java

```diff
@@ -46,28 +48,19 @@ public int getEnd() {
 
     /**
      * A span of elements.
-     * The syntax supported is given by the regexp: {@code "^([0-9]+)?(\\|[0-9]+)?"}
+     * The syntax supported is given by the regexp: {@code "^([0-9]+\\|[0-9]+)$"}
+     *
      * @param element a string that follows the syntax given by the regexp above
      * @return an element set with a "include from element (start)" to "last included element (end)"
      */
     public static Optional<Element> getElement(String element) {
-        if (null == element) {
-            return Optional.empty();
-        }
-        if (!element.matches("^([0-9]+)?(\\|[0-9]+)?")) {
```

Comment:
Wonder if the syntax of selecting just one element would most naturally be `1|1` or just `1` - it seems for some reason originally both the first part and the last part was optional. If the last part was kept optional you could still allow for a selection in its most simple form `1`

Reply:
I adjusted the code to allow both single element, and same start and end element.